tests/group/test_groups_absent.yml

```diff
@@ -0,0 +1,31 @@
+---
+- name: Include create_groups_json.yml
+  ansible.builtin.import_playbook: create_groups_json.yml
+
+- name: Test groups absent
+  hosts: ipaserver
+  gather_facts: false
+
+  tasks:
+  - name: Include groups.json
+    ansible.builtin.include_vars:
+      file: groups.json  # noqa 505
+
```

Comment:
Please insert this task:

    - name: Initialize groups_names
      ansible.builtin.set_fact:
        groups_names: []

This is needed if several of the groups tests using `groups_names` are executed in a row.